test/functional/test_framework/util.py

```diff
@@ -609,3 +610,13 @@ def sync_txindex(test_framework, node):
     sync_start = int(time.time())
     test_framework.wait_until(lambda: node.getindexinfo("txindex")["txindex"]["synced"])
     test_framework.log.debug(f"Synced in {time.time() - sync_start} seconds")
+
+def wallet_importprivkey(wallet_rpc, privkey, timestamp, *, label=""):
```

Comment:
Some function documentation here could prove to be helpful for others who are navigating the tests. Might even encourage to create more such helper/wrapper functions as needed.